server/impl/api-client-impl.js

```diff
@@ -86,22 +87,33 @@ Collection.prototype.getCollectionCacheKeys = function (publisherId, depth) {
   return { storyCacheKeys, collectionCacheKeys };
 };
 
-Collection.prototype.cacheKeys = function (publisherId, depth) {
-  if (!depth) {
-    return [collectionToCacheKey(publisherId, this)].concat([
-      ..._.flatMap(this.items, (item) => itemToCacheKey(publisherId, item)),
-      ...(this["collection-cache-keys"] ? this["collection-cache-keys"] : []),
-    ]);
+Collection.prototype.isAutomated = function() {
+  return !!this.automated;
```

Comment:
We'll have to verify if the the automated column or the data-source column is the source of truth for whether it is an automated collection or not. Checking this

Reply:
```
select * from story_collection_content where automated = false and data_source = 'automated' order by updated_at desc limit 100;

select * from story_collection_content where automated = true and data_source = 'manual' order by updated_at desc limit 100;
```

Both these queries return results and also have non-empty `rules`. :/

Reply:
I think the API is applying some logic over this instead of the actual db field.
https://bn.prothom-alo.com/api/v1/collections/potter-more -> returning `automated` as true even though the db value is false

https://bn.prothom-alo.com/api/v1/collections/fashion-botua -> returning `automated` as false though this result was got from the second sql query.